packages/apps/src/Endpoints/index.tsx

```diff
@@ -43,13 +44,21 @@ function isValidUrl (url: string): boolean {
   );
 }
 
+function getApiType (apiUrl: string): Endpoint {
+  if (apiUrl.includes('-substrate-connect')) {
+    return { param: apiUrl, type: 'substrate-connect' as EndpointType };
+  }
+
+  return { param: apiUrl, type: 'json-rpc' as EndpointType };
+}
```

Comment:
```suggestion
function getApiType (param: string): Endpoint {
  if (apiUrl.includes('-substrate-connect')) {
    return { param, type: 'substrate-connect' as EndpointType };
  }

  return { param, type: 'json-rpc' as EndpointType };
}
```

The casting `as EndpointType` should really not be needed. Can see why, but TS should be slightly more clever here...